tests/cold/test_ca_crt_migration.py

```
@@ -0,0 +1,152 @@
+    Kubernetes ca.crt convention. No migration is executed — the CA cert
+    code path (getTLSConfig -> Authenticate) runs during provider creation,
+    so plan readiness proves the feature works.
+    """
+
+    storage_map: StorageMap
+    network_map: NetworkMap
+    plan_resource: Plan
+
+    def test_verify_ca_crt_secret(
+        self,
+        prepared_plan: dict[str, Any],
```

Ruff (0.15.18)

[warning] Unused method argument: `prepared_plan`

(ARG002)